### ISSUE TITLE
Seperate TestSuite Results

### ISSUE THREAD

**Issue by sebake** [vmi/selenese-runner-java#209]
Dear Selenese-experts,
is there any possibility to split up the HTML result of very large TestSuites to different HTML Files for each test of the suite? 

Best regards

**Comment by vmutambuki** [comment]
Yes. We will contribute our solution that will hopefully replace the existing html. Additionally we have integrated, Selenese-runner into Bamboo CI for easy config. This splits big suites into individual test cases. https://marketplace.atlassian.com/plugins/com.mumosystems.charlotte/server/overview
